DDDEastAnglia.DatabaseMigrations/Migrations/3-AddSponsorsTable.cs

```diff
@@ -0,0 +1,25 @@
+﻿using FluentMigrator;
+
+namespace DDDEastAnglia.DatabaseMigrations.Migrations
+{
+    [Migration(20150411)]
+    public class AddSponsorsTable : Migration
+    {
+        public override void Up()
+        {
+            Create.Table("Sponsors")
+                .WithColumn("SponsorId").AsInt32().NotNullable().Identity().PrimaryKey()
+                .WithColumn("Name").AsString(50).NotNullable()
+                .WithColumn("Url").AsString(255).NotNullable()
+                .WithColumn("SponsorshipAmount").AsInt32().Nullable()
+                .WithColumn("Logo").AsBinary(int.MaxValue).NotNullable()
+                .WithColumn("PaymentDate").AsDateTime().Nullable()
+                .WithColumn("ShowPublicly").AsBoolean().NotNullable();
+        }
+
+        public override void Down()
+        {
+            Delete.Table("Sponsors");
```

Comment:
This might cause data loss, at worst in production if we have to roll back to the previous migration; at best when switching branches in dev/staging. Are we happy to accept this risk?